### DISCUSSION TITLE
Any plan to support the Bahamas?

### DISCUSSION THREAD

**Discussion by HercegAndras** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#355] [Q&A] [answered]
Hi Team
I just wondering, if this awesome hack working with the type **[Bahamas](https://m.bestwaycorp.com/Product/Detail/1060005XXX21)**?
Checked already the [instruction guide](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/blob/master/Build-instructions-Bestway-WiFi-remote.pdf), but it looks the Bahamas is not (yet) on the list,

many thanks!!
Andras

**Comment by visualapproach** [comment, marked as answer]
Hi @HercegAndras 
The pretty names on the spas is not significant in this case. Check instead the sticker on the pump for the model number.

**Reply by HercegAndras** [reply]
thanks! yep - indeed. supported:)
S100101 btw.

**Comment by Zolszabo** [comment]
Bahamas is 4 wire thing and with the single D1 mini approach that did not work for me.
